homeassistant/components/asuswrt/sensor.py

```diff
@@ -46,6 +63,18 @@ class AsusWrtSensorEntityDescription(SensorEntityDescription):
 
 UNIT_DEVICES = "Devices"
 
+CPU_SENSORS: tuple[AsusWrtSensorEntityDescription, ...] = tuple(
+    AsusWrtSensorEntityDescription(
+        key=sens_key,
+        translation_key=transl_key,
```

Comment:
We can use translation_placeholders here.

Reply:
Done